docs/pages/experiments/joy/list.tsx

```diff
@@ -716,7 +717,7 @@ const Gatsby = () => {
 export default function JoyList() {
   return (
     <CssVarsProvider
-      theme={{
+      theme={extendTheme({
```

Comment:
I like this change, as it is close to what we always had at MUI. A nit would be extracting this to a new const, but it's not too important for the experiments.